### PR TITLE
Feature 5387

### PR DESCRIPTION
**Pull request by Battula**
https://devops.dchbx.org/redmine/issues/5387
https://devops.dchbx.org/redmine/issues/7538
https://devops.dchbx.org/redmine/issues/7289
https://devops.dchbx.org/redmine/issues/7608

dependency on https://github.com/dchbx/enroll/pull/51

@raghuramg

### CODE REVIEW

**Comment by Battula** [comment]
Rebased on feature-7289 as it is dependent on the #7289

File: app/models/organization.rb [hunk 1]
```diff
@@ -203,6 +217,50 @@ def self.valid_carrier_names_for_options
...
+  # Expects file_path string with file_name format /hbxid_mmddyyyy_invoices_r.pdf
+  # Returns Date
+  def self.invoice_date(file_path)
+    date_string= File.basename(file_path).split("_")[1]
+    Date.strptime(date_string, "%m%d%Y")
+  end
+
+  def self.invoice_exist?(invoice_date,org)
+    documents =org.documents.where("date" => invoice_date)
+    binding.pry
```
Comment (Kevin-Wei) [inline app/models/organization.rb]: @Battula @raghuramg please review more carefully!

File: app/views/employers/premium_statements/show.html.erb [hunk 2]
```diff
@@ -45,7 +45,7 @@
     </table>
   </div>
 </div><br>
-<%= link_to 'Cancel', employers_employer_profile_path(@employer_profile.id, :tab=>'home'), class: 'btn btn-trans mz' %>
-  <a class='btn btn-trans' data-toggle='modal' data-target='#downloadCsv'>Download</a>
+<%= link_to 'Cancel', employers_employer_profile_path(@employer_profile.id, :tab=>'home'), class: 'btn btn-default mz' %>
+  <a class='btn btn-default' data-toggle='modal' data-target='#downloadCsv'>Download</a>
```
Comment (Kevin-Wei) [inline app/views/employers/premium_statements/show.html.erb]: 👍

**Comment by davidplappert** [comment]
@Battula Please fix merge conflicts

**Pull request merged**